package.json

```diff
@@ -32,7 +30,9 @@
         "format": "yarn format:js && yarn format:text",
         "format:js": "d2-style js apply",
         "format:text": "d2-style text apply",
-        "test": "jest"
+        "test": "d2-app-scripts test",
+        "storybook": "start-storybook -p 9009",
```

Comment:
It seems this isn't used, and is a duplicate of `yarn start:demo`, can we remove it?

Reply:
Yeah, Storybook CLI re-added it when I was trying to figure out what was wrong with our setup. Removed in e6e370f.